kivy/modules/touchring.py

```diff
@@ -15,6 +15,15 @@
         Scale of the image.
     `alpha`: float, defaults to 1.
         Opacity of the image.
+    `show_cursor`: boolean, default to False
```

Comment:
@kenwaldek this isn't really a good place to discuss it :D The PR is old and github is for issues, so instead - read [Community Guidelines](https://github.com/kivy/kivy/wiki/Community-Guidelines) and come visit our channels :)